README.md

```diff
@@ -22,23 +22,4 @@ And for web service application you can use as a parent POM :
     <version>${farao.parent.version}</version>
 </parent>
 ```
-It inherits from farao-parent so it will embed the same modules. In addition it uses spring-boot-dependencies-bom in version 5.6.3. Then the main use case will be to build spring boot application in FARAO context. 
-
-Additionally two BOMs are defined: powsybl-bom and farao-bom. It defines versions for respectively PowSyBl and FARAO modules. FARAO BOM includes PowSyBl BOM.
-
-```
-<dependency>
-    <groupId>com.farao-community.farao</groupId>
-    <artifactId>powsybl-bom</artifactId>
-    <version>${farao.parent.version}</version>
-    <scope>import</scope>
-    <type>pom</type>
-</dependency>
-<dependency>
-    <groupId>com.farao-community.farao</groupId>
-    <artifactId>farao-bom</artifactId>
-    <version>${farao.parent.version}</version>
-    <scope>import</scope>
-    <type>pom</type>
-</dependency>
-```
+It inherits from farao-parent so it will embed the same modules. In addition it uses spring-boot-dependencies-bom. Then the main use case will be to build spring boot application in FARAO context. 
```

Comment:
imports instead of uses and you can put spring-boot-dependencies (more accurate) and spring-cloud-dependencies